index.js

```diff
@@ -37,27 +38,42 @@ const format = (markup, url) => {
 
   const api = [];
 
-  console.log(`\n${chalk.bold(title)}\n`);
-  console.log(wrap(description));
-  console.log(`\n${usage}\n`);
-
-  $('#wikiArticle dl dt').each((index, element) => {
-    const $element = $(element);
-    const term = $element.text();
-    const definition = $element
-      .next('dd')
-      .text()
-      .replace(new RegExp(term, 'gim'), chalk.bold(term));
-
-    api.push({
-      term: chalk.bold(term),
-      definition
+  console.log(`\n${chalk.bold(title)}`);
+  console.log(`\n${wrap(description)}\n`);
+
+  if (/[a-z]/.test(usage)) {
+    console.log(`${usage}\n`);
+  }
+
+  $('#wikiArticle dl dt')
+    .has('code')
+    .each((index, element) => {
+      const $element = $(element);
+      const term = $element.text();
+      const definition = $element
+        .next('dd')
+        .text()
+        .replace(new RegExp(term, 'gim'), chalk.bold(term));
+
+      api.push({
+        term: chalk.bold(term),
+        definition
+      });
+
+      api.push({term: '', definition: ''});
     });
 
-    api.push({term: '', definition: ''});
+  const cleanApi = api.map(apiObject => {
```

Comment:
It was simply the same as api, but cleaned up as it has the duplicated information removed. The only times information is duplicated is when apiObject.term is "callback" (for example, check the output for `mdn array.map`). I agree it's not at all the most elegant solution to the problem though, it was really just a quick fix as I figured you want to reformat it later anyway.

Ideally, the output would be something like this:
```
callback               Function that produces an element of the new Array, taking three arguments:
    currentValue    The current element being processed in the array.
    index                The index of the current element being processed in the array.
    array                The array map was called upon.
thisArg                 Optional. Value to use as this when executing callback. Default value is the Window object
```